gensim/models/wrappers/fasttext.py

```diff
@@ -123,11 +123,8 @@ def __contains__(self, word):
         if word in self.vocab:
             return True
         else:
-            word_ngrams = set(FastText.compute_ngrams(word, self.min_n, self.max_n))
-            if len(word_ngrams & set(self.ngrams.keys())):
-                return True
-            else:
-                return False
+            word_ngrams = FastText.compute_ngrams(word, self.min_n, self.max_n)
+            return any(ng in self.ngrams for ng in word_ngrams)
```

Comment:
`compute_ngrams` previously used to return a set of unique ngrams. The reason this was changed was that the original FastText implementation sums up vectors for _all_ character ngrams to obtain a word vector (so ngrams occurring more than once are actually added up multiple times), and so we wanted to replicate FastText behaviour as closely as possible.  

An iterator would probably be ideal, and the char ngrams are already returned in increasing order of length, so the `any(ng in self.ngrams for ng in word_ngrams)` should be fairly efficient. Returning them in decreasing order of frequency would be non-trivial though (partly because we don't even store ngram counts anyway), and probably unnecessary.

I'm not sure why the previous construct was the way it was - I can't think/remember of any good reason for it. There is already a very similar snippet in the `word_vec` method - 
```python
ngrams = FastText.compute_ngrams(word, self.min_n, self.max_n)
ngrams = [ng for ng in ngrams if ng in self.ngrams]
```
So I'm not sure why I didn't already stick to that. Thanks a lot for the fix.